src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.OpenSsl.cs

```diff
@@ -26,6 +26,21 @@ internal static partial class OpenSsl
         private unsafe static readonly Ssl.SslCtxSetAlpnCallback s_alpnServerCallback = AlpnServerSelectCallback;
         private static readonly IdnMapping s_idnMapping = new IdnMapping();
 
+        static OpenSsl()
```

Comment:
Before this change, the compiler could make the Interop.OpenSsl class beforefieldinit.  Adding this explicit static cctor is going to stop that from happening, which can have perf implications.